specs/2025-02-24.yaml

```diff
@@ -1023,199 +938,635 @@ paths:
+          example: "Animal Welfare"
+        parent_code:
+          type: string
+          description: The parent code of the NTEE code
+          example: "A"
+        parent_description:
+          type: string
+          description: The description of the parent NTEE code
+          example: "Animal Welfare"
+    NaicsCode:
+      type: object
+      description: |-
+        A code that is used to classify the nonprofit entity as defined by the NAICS.
+      properties:
+        naics2:
+          type: string
+          description: The code of the NAICS code
+          example: "31"
+        naics2_description:
+          type: string
+          description: The description of the NAICS code
+          example: "Manufacturing"
+        naics4:
+          type: string
+          description: The code of the NAICS code
+          example: "3112"
+        naics4_description:
+          type: string
+          description: The description of the NAICS code
+          example: "Grain and Oilseed Milling"
+        naics6:
+          type: string
+          description: The code of the NAICS code
+          example: "311221"
+        naics6_description:
+          type: string
+          description: The description of the NAICS code
+          example: "Wet Corn Milling and Starch Manufacturing"
+    ComplianceRequirement:
+      type: object
+      description: |-
+        A compliance requirement is a requirement that a nonprofit entity must meet in order to be eligible for Chariot services.
+      properties:
+        compliant:
+          type: boolean
+          description: Whether the entity is compliant with the requirement
+          example: true
+        reason:
+          type: string
+          description: The reason for the compliance requirement
+        last_found_at:
+          type: string
+          format: date-time
+          description: The date and time when the compliance requirement was last found
+          example: "2020-07-10 15:00:00.000"
+    WebDomain:
+      type: object
+      description: |-
+        A web domain is a unique identifier for a website or web application that is associated with an organization.
+        Domains are important identity constructs that underpin the web and online interactions.
+        The domain can be used as an identifier and its validity can be asserted via DNS.
+      required:
+        - domain
+      properties:
+        domain:
+          type: string
+          description: The DNS domain name
+          example: "redcross.org"
+    Brand:
+      type: object
+      description: |-
+        Fair use brand assets that are associated with an organization and can be used for identification and informational purposes.
+      properties:
+        icon_url:
+          type: string
+          description: The URL of the organization's icon
+        logo_url:
+          type: string
+          description: The URL of the organization's logo
```

Comment:
we store these internally as arrays. Do we want these to be arrays too? Or just return the first item in our array?